.github/workflows/pull_request.yml

```diff
@@ -6,6 +6,7 @@ on:
 
 jobs:
   verify:
+    if: ${{ !startsWith(github.head_ref, 'dependabot/')  }}
```

Comment:
Will this not run the checks on Dependabot PRs?

Reply:
If not, I can understand why we wouldn't run linting, formatting etc, but I think it's important to run the build to make sure nothing breaks with the updates. May have misunderstood though :)

Reply:
That's right, there is no need to waste github actions minutes since the workflow will run on the new combined PR

Reply:
Ah I see, so the combined PR will run them, just not the individual Dependabot ones.

Can we prevent the Dependabot ones from being merged since they can't be verified the same as others? Or does that happen anyway due to branch protection rules?

Reply:
Yes, that happens automatically since they will not be passing the `verify` stage which is required to be able to merge.